### ISSUE TITLE
Transfer and modify content from ECP "Getting Started" folder into Cookbook

### ISSUE THREAD

**Issue by alexishunzinger** [NASA-Openscapes/earthdata-cloud-cookbook#286]
A lot of work was completed for the Earthdata Cloud Playground effort, which was rejected, but is transferrable to the Cookbook!

We will transfer content found in the ["Getting Started" folder from the nasa/Earthdata-cloud-playground repository](https://github.com/nasa/Earthdata-cloud-playground/tree/main/1-Getting-Started) and modify it to fit into the Earthdata Cloud Cookbook.

This requires reviewing any duplicate information and adjusting verbiage to refer to the Cookbook of 2i2c Openscapes JupyterHub.

**Comment by alexishunzinger** [comment]
Most of the content in Getting Started is specific to a JupyterHub. Do we have a place where we layout JupyterHub guidance? Cookbook or elsehwere?

**Comment by jules32** [comment]
Hi @alexishunzinger would this be the place for that? https://nasa-openscapes.github.io/earthdata-cloud-cookbook/environment-setup/ 

I know we had wanted to revisit this page to describe the corn(+) environment so we could link to JupyterHub guidance there? I think it's well documented by 2i2c and elsewhere. 

Also please disregard if off-base, I know you have been deep into this and it's been awhile for me :)

**Comment by alexishunzinger** [comment]
Yeah I think the Cloud Environment Setup page will get way beefier. There are a few ways to get "into the cloud" and we have tons of info relevant to JupyterHubs

**Comment by alexishunzinger** [comment]
@alexishunzinger Revisit this with the Cloud Playground materials OR have a discussion about what we want to include here
